frontend/src/components/project/ProjectContent.js

```diff
@@ -357,3 +401,43 @@ function CollaborateContent({ project, texts }) {
     </>
   );
 }
+
+function DiscussionPreview({ latestParentComment, discussionTabLabel, locale, project }) {
+  const classes = useStyles();
+  return (
+    <>
+      <Link
+        href={`${getLocalePrefix(locale)}/projects/${project.url_slug}#comments`}
+        underline="none"
+      >
+        <div className={classes.discussionPreview}>
+          <div className={classes.topSectionDiscussionPreview}>
+            <Typography
+              display="inline"
+              color="primary"
+              className={classes.headingDiscussionPreview}
+            >
+              {discussionTabLabel}
+            </Typography>
+            <UnfoldMoreIcon color="secondary" className={classes.iconDiscussionPreview} />
+          </div>
+          <div className={classes.commentDiscussionPreview}>
+            <Avatar
+              className={classes.commentAvatarDiscussionPreview}
+              src={
+                latestParentComment.author_user.image
+                  ? getImageUrl(latestParentComment.author_user.image)
+                  : getImageUrl(latestParentComment.author_user.thumbnail_image)
+              }
+            />
+            <Typography className={classes.commentTextDiscussionPreview}>
+              <Truncate lines={3} ellipsis={"..."}>
+                {latestParentComment.content}
+              </Truncate>
+            </Typography>
+          </div>
+        </div>
+      </Link>
+    </>
+  );
+}
```

Comment:
I would create a new file for this

Reply:
The DiscussionPreview-function has shrunk quite a bit since using the post component. Do you still prefer to put it in its own file?

Reply:
Yeah I think that makes sense since it's logically it's own component

Reply:
Alright, it's done